docs/api-reference.md

```
@@ -0,0 +1,222 @@
+# WandererKills API Reference
+
+## Base URL
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)